src/User/UserContainer.jsx

```diff
@@ -2,46 +2,52 @@ import React, { useState, useEffect } from "react";
 import { Switch, Route, useParams } from "react-router-dom";
 import { withCookies, useCookies } from "react-cookie";
 import { cookieName } from "../Constants/Cookies";
-import { getUserById } from "../api.js";
 import ProfilePage from "./ProfilePage";
 import UpdateProfile from "./UpdateProfile";
+import { getUserById } from "../api.js";
 
-const UserContainer = () => {
-  const [user, setUser] = useState(null);
-  const [isLoading, setIsLoading] = useState(true);
-  const [error, setError] = useState(false);
-  const { id } = useParams();
-  const [cookies] = useCookies([cookieName]);
+const UserContainer = ({ sessionToken }) => {
+	const [user, setUser] = useState(null);
+	const [isLoading, setIsLoading] = useState(true);
+	const [error, setError] = useState(false);
+	const { id } = useParams();
+	const [cookies] = useCookies([cookieName]);
```

Comment:
you're using hooks and a higher order component at the bottom too, the higher ordered component will inject `userContainer` with the props allCookies & cookies where as the hooks access the cookies directly. if you're going with the hook approach please remove the withCookies around the export, and you don't need to pass in `sessionToken` as props as you have access to cookies here so it'll be the same as `cookies.sessionToken`

Reply:
Hi Tony, are you free at all to go through this today? I have some questions

Reply:
Sure, have a read of this might help explain the difference https://medium.com/javascript-scene/do-react-hooks-replace-higher-order-components-hocs-7ae4a08b7b58